lib/registration.pm

```diff
@@ -219,7 +220,12 @@ sub fill_in_registration_data {
                 }
                 if (my $regcode = get_var("SCC_REGCODE_$uc_addon")) {
                     # skip addons which doesn't need to input scc code
-                    next unless grep { $addon eq $_ } qw(ha geo we live rt ltss phub);
+                    if (sle_version_at_least('15')) {
+                        next unless grep { $addon eq $_ } qw(ha geo live rt ltss phub we);
+                    }
+                    else {
+                        next unless grep { $addon eq $_ } qw(ha geo live rt ltss);
+                    }
```

Comment:
I suggest to conditionally add entries to the list instead, e.g.

```perl
my @modules_without_regcode = qw(ha geo live rt ltss);
@modules_without_regcode .= qw(phub we) if sle_version_at_least('15');
```

but I wonder why you changed phub as well.

Reply:
oops.

Reply:
But Olli, you understood it wrong, it is the other way round.